### PR TITLE
Consumer change email

### PR DESCRIPTION
**Pull request by c3zi**

### CODE REVIEW

File: src/UseCase/SendChangeUserEmailConfirmation.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+     * @var string
+     */
+    private $subject;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    public function __construct(
```
Comment (adshares-tech) [inline src/UseCase/SendChangeUserEmailConfirmation.php]: MAJOR This function has 8 parameters, which is greater than the 7 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS107)

**Comment by adshares-tech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: .env.dist [hunk 2]
```diff
@@ -38,3 +38,9 @@ QUEUE_HOST=adshares_ads-operator_queue_1
 QUEUE_PORT=5672
 QUEUE_USER=guest
 QUEUE_PASSWORD=guest
+
+MAILER_URL=null://localhost
```
Comment (pawzar) [inline .env.dist]: `null`?

File: config/packages/dev/swiftmailer.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+swiftmailer:
+    delivery_addresses: ['mailcatcher@adshares.net']
```
Comment (pawzar) [inline config/packages/dev/swiftmailer.yaml]: `env` ?

File: .env.dist (hunk 2 above)
Comment (c3zi) [reply .env.dist]: Yep, default solution to disable email delivery. It was created by Symfony Flex. It has to be changed (ALWAYS) on the prod.

Comment (c3zi) [reply .env.dist]: https://symfony.com/doc/current/email.html

**Pull request merged**